src/document/DocumentAnnotator.ts

```diff
@@ -10,6 +10,11 @@ import { Mode } from '../store';
 import { scrollToLocation } from '../utils/scroll';
 import './DocumentAnnotator.scss';
 
+export const ANNOTATION_CLASSES: { [M in Mode]?: string } = {
+    [Mode.HIGHLIGHT]: 'ba-create--highlight',
+    [Mode.REGION]: 'ba-create--region',
```

Comment:
`.ba-is-create--region` and `.ba-is-create--highlight`?

Reply:
Good catch, will fix.